#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_siputils.dpatch by Torsten Marek <shlomme@debian.org>
Index: sip4-qt3-4.10/siputils.py
===================================================================
--- sip4-qt3-4.10.orig/siputils.py	2010-01-14 15:52:09.000000000 +0100
+++ sip4-qt3-4.10/siputils.py	2010-01-26 21:09:49.172700039 +0100
@@ -718,7 +718,8 @@
 
         libs.extend(self.optional_list("LIBS_WINDOWS"))
 
-        lflags.extend(self._platform_rpaths(rpaths.as_list()))
+        # Don't append any rpaths
+        #lflags.extend(self._platform_rpaths(rpaths.as_list()))
 
         # Save the transformed values.
         self.CFLAGS.set(cflags)
@@ -843,6 +844,15 @@
         clib is the library name in cannonical form.
         framework is set of the library is implemented as a MacOS framework.
         """
+        ##################################################################
+        # Generally, the linker is intelligent enough not to need this   #
+        # additional information!                                        #
+        # And Qt4's pkg-config and prl files are broken                  #
+        # Changed for Debian packaging, Torsten Marek <shlomme@gmx.net>  #
+        ##################################################################
+
+        return []
+        
         prl_libs = []
 
         if self.generator in ("MSVC", "MSVC.NET", "BMAKE"):
